README.md

```
@@ -1,101 +1,91 @@
+This story-based chatbot follows the "Sogang Campus Survival Guide" of Alos, who has descended to campus to live as a human.
+From an omniscient senior's perspective, users will guide Alos through his choices, experiencing growth and conflict together.
 
-## 3. generation/chatbot@.py
+---
 
-### 응답 및 임베딩 비교 코드:
+## 📱 Screenshots
 
-- OpenAI API를 활용한 답변 생성과, 임베딩을 통한 유사도 계산(또는 RAG 등) 로직을 모두 이 파일에 작성합니다.
+<p align="center">
+  <img src="https://github.com/user-attachments/assets/ec89d711-9ec4-4095-b50d-05db3541813d" width="500"/>
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)